### ISSUE TITLE
How should I replace my own dataset for the experiment？

### ISSUE THREAD

**Issue by weican1103** [t9liverpool/OpenMix#1]
Cool, I find your work to be very fascinating. I am currently in the process of revising a paper and need to add a comparative experiment. My dataset is in the .mat format (train.mat and test.mat), and it consists of 11 categories. For each category being considered as unknown, there is a corresponding set of train and test data; for instance, when the 0th category is unknown, there is one set of train and test data, and when the 1st category is unknown, there is another set of train and test data. I am interested in knowing if I can effectively integrate my own dataset within your framework. Secondly, I would like to understand if it's possible to appropriately customize the labels for known and unknown categories within the code.

**Comment by t9liverpool** [comment]
Hi, 
#1: The effectiveness of this algorithm has been evaluated under several natural image datasets.  If the OSR performance of the softmax method is not saturated on your dataset, and if the semantics of images are sensitive to rotation transformation, there is a good chance to make it work. However, if the foreground has a circle-like shape, be cautious to use OpenMixup. If the background is tedious, saying, plain black, be cautious to use OpenCutout. 

By the way, you might transform your image matrices into tensors and have a try. I guess there are several built-in functions in Python that support this operation. 

#2: You might check out torch.utils.data.DataLoader, and you can upload your customized scripts via collate_fn.
